.github/workflows/ci.yaml

```diff
@@ -167,6 +165,14 @@ jobs:
             os_version: "16.0"
           - toxenv: kernel_module
             os_version: "16.0"
+          - toxenv: all
+            os_version: "15.7"
```

Comment:
although don't have a strong opinion, I think sorting these before the "16.0" section would be slightly more logical.